### ISSUE TITLE
feat (model): 1 fit and predict on the training dataset

### ISSUE THREAD

**Issue by ezalos** [42-AI/SentimentalBB#20]
# 📖 Describe what you want

While waiting for the allocine training dataset from the [Milestone 3](https://github.com/42-AI/SentimentalBB/milestone/3) let's train a model with our temporary dataset.

The temporary dataset is the `aclImdb` Standford datasaet.
 - It can be download with `sh scripts/download_dataset.sh`
 - The training dataset is then constructed by running `python -m src data --download aclImdb`

The 1st model:
 - You should fit() and predict() a sklearn classifier (for example Naive Bayes) on this train dataset.

# ✔️ Definition of done

 When you have your predicted results saved under `data/processed/aclImdb/results/classifier_name.csv`
 
 Your results should at least contain: `y_pred` and `y_true` (as we are using the training dataset, we have access to the correct label associated with each X) 
 
 
There should should be a unit test which launches the train and predict pipeline, and then calculate the accuracy from this saved file.
